README.md

```diff
@@ -16,11 +16,18 @@ We plan to update the simulator as more and more about COVID-19 will be known.
 
 
 ## Dependencies
-Following `python` packages are required (python>=3.7)
+To install the simulator without `ctt`
 ```
-pip install -r requirements.txt
+pip install covid19sim
```

Comment:
I think for now it should read `pip install .`

Reply:
Ok! thanks for pointing that out. fixed it!